web/client/codechecker_client/cmd_line_client.py

```diff
@@ -881,6 +881,21 @@ def get_diff_local_dir_remote_run(
         local_report_hashes = set([r.report_hash for r in report_dir_results])
         local_report_hashes.update(baseline.get_report_hashes(baseline_files))
 
+        review_status_filter = ttypes.ReviewStatusRuleFilter()
+        review_status_filter.reportHashes = local_report_hashes
+        review_status_filter.reviewStatuses = [
+            ttypes.ReviewStatus.FALSE_POSITIVE,
+            ttypes.ReviewStatus.INTENTIONAL]
+        closed_hashes = set(map(
+            lambda x: x.reportHash,
+            client.getReviewStatusRules(
+                review_status_filter, None, None, None)))
+
+        local_report_hashes -= closed_hashes
+        report_dir_results = [
```

Comment:
No, it's because we subtract all closed reports from the ones on client side. This was the last missing feature from this functionality: in a remote-local diff we consider local reports as closed if they are closed remotely. Should we distinguish "open report because it's unreviewed" and "open report because it's confirmed" and subtract only from the first group? :)

Reply:
My problem is that I marked a report with source code comment as Confirmed (so it's a real bug), and altough the same report hash already has a review status rule but it doesn't affect this report because the source code comment is stronger. So the analyzer found this problem and the user already confirmed that this is a real bug but we hide it.

I think the situation is the same when there is no source code comment for a report and there is no review status rule for this report hash. In this case you will show this report with one of the diff option as new/resolved/unresolved.